src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/SmtpConfigurationType.php

```diff
@@ -37,49 +38,73 @@
  */
 class SmtpConfigurationType extends TranslatorAwareType
 {
+    public const FIELD_MAIL_DOMAIN = 'domain';
+    public const FIELD_MAIL_SERVER = 'server';
+    public const FIELD_MAIL_USER = 'username';
+    public const FIELD_MAIL_PASSWD = 'password';
+    public const FIELD_MAIL_SMTP_ENCRYPTION = 'encryption';
+    public const FIELD_MAIL_SMTP_PORT = 'port';
+
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('domain', TextType::class, [
+            ->add(self::FIELD_MAIL_DOMAIN, TextType::class, [
```

Comment:
```suggestion
            ->add(self::FIELD_MAIL_DOMAIN, TextType::class, [
```

Replace `self` by `static`.